--- AcelleClient_original.php	2021-03-03 16:20:00.072272921 -0300
+++ AcelleClient_editado.php	2021-03-03 16:19:54.024224612 -0300
@@ -197,7 +197,7 @@
             unset($args['api_token']);
         }
         $query['api_token'] = !empty($query['api_token']) ? $query['api_token'] : $this->getToken();
-        $url = '/lists/'.$list_uid.'/subscribers/store'.'?'.http_build_query($query);
+        $url = '/subscribers'.'?'.'list_uid='.rawurlencode($list_uid).'&'.http_build_query($query);
         $headers = [
             'Accept: application/json',
             'Content-Type: application/json',
@@ -238,7 +238,7 @@
             unset($args['api_token']);
         }
         $query['api_token'] = !empty($query['api_token']) ? $query['api_token'] : $this->getToken();
-        $url = '/lists/'.$list_uid.'/subscribers/'.$uid.'/update'.'?'.http_build_query($query);
+        $url = '/subscribers/'.$uid.'?'.http_build_query($query);
         $headers = [
             'Accept: application/json',
             'Content-Type: application/json',
