--- autoptimizeImages_original.php	2020-12-15 23:22:00.000000000 -0300
+++ autoptimizeImages_editado.php	2021-02-11 18:57:09.163581198 -0300
@@ -770,6 +770,7 @@
     public function add_lazyload( $tag, $placeholder = '' ) {
         // adds actual lazyload-attributes to an image node.
         $this->lazyload_counter++;
+        $this->options['autoptimize_imgopt_number_field_7'] = isset($this->options['autoptimize_imgopt_number_field_7']) ? $this->options['autoptimize_imgopt_number_field_7'] : '0';
         $_lazyload_from_nth = apply_filters( 'autoptimize_filter_imgopt_lazyload_from_nth', $this->options['autoptimize_imgopt_number_field_7'] );
         if ( str_ireplace( $this->get_lazyload_exclusions(), '', $tag ) === $tag && $this->lazyload_counter >= $_lazyload_from_nth ) {
             $tag = $this->maybe_fix_missing_quotes( $tag );
